src/Components/test/E2ETest/Tests/EventTest.cs

```diff
@@ -155,8 +155,6 @@ public void MouseDownAndMouseUp_CanTrigger()
         var output = Browser.Exists(By.Id("output"));
         Assert.Equal(string.Empty, output.Text);
 
-        var other = Browser.Exists(By.Id("other"));
-
```

Comment:
This removal is unrelated to the fix - I just noticed that the `other` variable was unused and that the test appears to otherwise not care about the existence of the `id="other"` element.